src/defaults.js

```diff
@@ -89,101 +100,48 @@ const DEFAULT_SET_JSON_OPTIONS = {
 };
 
 /**
- * URI_SKYNET_PREFIX.
+ * The raw size in bytes of the data that gets put into a link.
  */
-const URI_SKYNET_PREFIX = "sia://";
-
-const JSON_RESPONSE_VERSION = 2;
+const RAW_SKYLINK_SIZE = 34;
```

Comment:
Some of these are not really defaults but rather constants. Can you put them in a more suitable file?

Reply:
Hello m-cat,

then I didn't understand the difference.
I have left everything in "src/defaults.js" that I consider to be Skynet's defaults.

Now I'm even more unsure.
That means I need precise information on what to change.